src/main/java/appeng/me/cluster/implementations/CraftingCPUCluster.java

```diff
@@ -1569,6 +1580,34 @@ public int getRemainingOperations() {
         }
     }
 
+    public void tryExtractItems() {
```

Comment:
This is a pretty expensive check, have you tested if this causes lag on a large craft?

Reply:
This isn't the most thorough test but considering these profiles are basically identical I think this is a reasonable test